text/game/abilities.stringtable

```diff
@@ -4471,7 +4471,7 @@
     </Entry>
     <Entry>
       <ID>911</ID>
-      <DefaultText>Weist den tierischen Gefährten des Waldläufers an, zu einem Ziel zu preschen und es Niedergeschlagen wird.</DefaultText>
+      <DefaultText>Weist den tierischen Gefährten des Waldläufers an, zu einem Ziel zu preschen und es niederzuwerfen.</DefaultText>
```

Comment:
Oder wird Niedergeschlagen nochmal in der Zusammenfassung der Fähigkeit erwähnt? Dann bin ich mit korrekter Grammatik in der Beschreibung einverstanden. :)